<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonnelReportWindow</class>
 <widget class="QMainWindow" name="PersonnelReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчет по персоналу</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnBack">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSearch">
        <property name="text">
         <string>Поиск:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSearch">
        <property name="placeholderText">
         <string>Введите ФИО или специальность</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGender">
        <property name="text">
         <string>Пол:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbGender"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
